Articles of Association

Defman opened the pull request.

Defman committed:
init

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,66 @@
+    1. The name of the association is "Feather Association"
+2. Purpose
+    1. The purpose of the association is
+        1. The promotion of feather
```
Might wanna further specify what we mean by "feather"

caelunshun replied:
We'd want to include Feather as well as all its subprojects in this definition—i.e, the organization as a whole.

Defman replied:
Do we want to include all subprojects in the purpose? Would that not be included in the general development of feather? Would the GitHub organization and its repositories not be an asset to the association? We should maybe also add in support for general Minecraft modding. Since we may also want to support projects like https://wiki.vg verbally.

caelunshun commented on Articles of Association.md:
```diff
@@ -0,0 +1,66 @@
+    3. Paying Members pay a membership fee determined by the general meeting.
+4. General Assembly
+    1. The general meeting is the highest authority of the association.
+    2. The Annual General Meeting is held once a year before the end of the ______ month and is convened with at least three weeks' notice by email to members who are eligible to vote.
```
General Meeting or General Assembly?

Defman replied:
General Assembly, my bad.

caelunshun commented on Articles of Association.md:
```diff
@@ -0,0 +1,66 @@
+        12. TODO: Optionally/Any subject not on the agenda
+    6. Proposals that are to be considered at the General Assembly must be received by the board of directors no later than 8 days before the General Assembly. Proposals for amendments to the articles of association that are to be considered must be received by the Board of Directors no later than three weeks before the General Assembly.
+    7. The General Assembly is chaired by a chairman who cannot be a member of the board.
+    8. The General Assembly makes its resolutions by 1/2 of votes and by public written voting.
```
`public written voting` => `Votes are public`? Or is that not what this means?

Defman replied:
Votes could be done by raise of hands... which for obvious reason would be hard. That's why I specified that votes are cast by writing. I don't know if this is the best way to express it, we would like to keep it vague, such that we can choose our own voting method.

caelunshun commented on Articles of Association.md:
```diff
@@ -0,0 +1,66 @@
+        1. The promotion of feather
+        2. The development of feather
+        3. The distribution of feather and associated software and tools
+3. Membership
```
I don't see anything about how one becomes a voting member.

Defman replied:
Approval by other voting members, doing the General Assembly.

Defman replied:
I have made it possible for any member to be vote eligible if they are active.

Defman commented:
There are pros for making it easy to become a member and have voting be restricted to a subgroup of members. But there are also pros for making it a tad harder to become a member and simply have all member vote at the General Assembly. In both cases, we can accept donations, but member statistics are usually important for associations and therefor we may prefer the first option.

Defman committed:
amendments

Defman committed:
amendments

Defman commented on Articles of Association.md:
```diff
@@ -0,0 +1,66 @@
+    1. The general meeting is the highest authority of the association.
+    2. The Annual General Meeting is held once a year before the end of the ______ month and is convened with at least three weeks' notice by email to members who are eligible to vote.
+    3. The public is eligible to attend the general meeting.
+    4. All members who are actively contributing towards the goals stipulated in section 2. and has notified the Board of Directors 8 days prior to the General Assembly are eligible to vote.
```
This way only people who have contacted the Board of Directors 8 days prior are eligible to vote. Maybe we should shorten the time required and make board members automatically have voting rights or do they automatically contact themself? Also, it is kinda authoritarian having the Board of Directors "decide" who gets to vote. Maybe we should set some limit for how many can be eligible to vote?

Defman reviewed (commented):
I have removed the concept of Paid Members and Voting Members; instead, I made all active members vote eligible with some formality.

Defman commented:
We should maybe also make it clear that the association is non-profit, maybe change the purpose to "support the development of the open-source Minecraft server Feather and associated tools". I don't know if we should have an article stating "this association is strictly non-profit".

Defman committed:
merge

Defman committed:
amendments and proposal for initial policies

Defman committed:
spelling

Defman committed:
spelling

Defman committed:
budget

Defman committed:
changes to the budget

Defman committed:
yearly budget

Defman committed:
specified that the General Assembly is held electronicly

Defman committed:
stuff

Defman committed:
spelling

Defman committed:
spelling

Defman committed:
preparations for founding

Defman committed:
signing of the summary

Defman committed:
updated the agenda

Defman committed:
removed vote counter, it is the job of the speaker

Defman committed:
vice-chairman -> deputy chairman

PauMAVA committed:
Update speaker.md

Grammar and some punctuation errors. I think it's better if we talk about `The Speaker` instead of `you`.

Defman committed:
Merge pull request #1 from PauMAVA/patch-1

Update speaker.md

Defman committed:
deputy members -> substitues

Defman committed:
added a description of the association

Defman committed:
merge

Defman committed:
initial workgroups

Defman committed:
cvr link to english version

Defman committed:
feather workgroup

Defman committed:
added maintance to the purpose

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,57 @@
+6. The daily management of the association
+    1. The daily management of the association consists of the board, which in addition to the chairman consists of the deputy chairman, the treasurer and two to six additional members. The board of directors is elected by the General Assembly.
+    2. The board of directors leads the association in accordance with these articles of association and the resolutions of the General Assembly.
+    3. No later than 14 days after re-electing the Board of Directors, a constituent meeting is held, where the Board of Directors elects the treasurer from among its members. All positions of trust are valid until the next General Assembly.
```
"All positions of trust are valid until the next General Assembly." It might be wise to allow for a re-election of the board of directors at an Extraordinary General Assembly?

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,57 @@
+8. Signing authority
+    1. The association is signed externally by the signature of the chairman and at least one additional board member in the association.
+    2. The members of the association are not personally liable for the obligations incumbent on the association.
+9. Amendments to the Articles of Association
```
Probably should be changes in general and not only amendments, right?

Schuwi replied:
Ah, nevermind, misunderstood the definition of amendment

Defman committed:
amendments to 6.3. and updated the index of workgroups

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,57 @@
+    1. The General Assembly is the highest authority of the association.
+    2. The Annual General Assembly is held electronicly once a year before the end of ______ and is convened with at least three weeks' notice to members.
+    3. The public is eligible to attend the General Assembly.
+    4. All members who are actively contributing towards the purpose stipulated in section 2 are eligible to vote.
```
how would we evaluate, whether a member "actively contributes towards the purposes". is this necessary?

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,57 @@
+        7. Approval of the budget
+        8. Election of chairman
+        9. Election of deputy chairman
+        10. Election of 3-7 board members and 2 substitutes
```
I am still not quite sure why we need 2 substitutes rather than just increasing the number of board members to 5-9 if really necessary

Defman replied:
Board members are active, while substitutes can stay inactive. Like, I don't want to be a board member, but if you guys a missing one, count me in. If someone leaves the board and we are stile above 5 substitutes can decline without requiring an extraordinary assembly.

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,57 @@
+    3. The association's accounts are kept by the treasurer, who also keeps the association's membership register.
+    4. The accounts are audited by the auditor (s) and deputy auditor elected at the General Assembly.
+8. Signing authority
+    1. The association is signed externally by the signature of the chairman and at least one additional board member in the association.
```
"signed externally" is still very weird wording but I am not quite sure how to improve that;

also the deputy chairman should probably be allowed in place of the chairman in case of absence right?

Defman replied:
That the deputy chairman can stand in for the chairman is a given.

Schuwi commented on Articles of Association.md:
```diff
@@ -0,0 +1,57 @@
+    2. The members of the association are not personally liable for the obligations incumbent on the association.
+9. Amendments to the Articles of Association
+    1. These Articles of Association can only be amended by a 2/3 majority at a General Assembly, where the proposed amendment appears on the agenda.
+    2. The amendments to the articles of association enter into force with effect from the General Assembly at which they are adopted.
```
so, **after** the General Assembly at which they are adopted?

Schuwi committed:
Improve the wording of the Articles of Association

Defman committed:
Update Articles of Association.md

Defman committed:
Update Articles of Association.md

Defman committed:
Merge pull request #2 from Schuwi/patch-1

Improve the wording of the Articles of Association

Defman committed:
specified notice by email

Defman committed:
merge

Defman committed:
changes

Defman committed:
changes

Defman committed:
yearly assembly by the end of April

Defman committed:
name change

Defman committed:
updated the budget

Defman committed:
updated the budget

Defman committed:
updated the budget

Defman committed:
updated the budget

Defman committed:
updated the budget

Defman committed:
meeting
